Cron_Philip/Assignments/JSfundP3/JSfundP3.js

```diff
@@ -0,0 +1,73 @@
+function personConstructor(name) {
+
+  var simPerson = {
+    name: name,
+    distance_traveled: 0,
+    say_name: function(){
+      console.log(this.name)
+    },
+    say_something: function(){
+      console.log(this.name, "says I am so cool!" )
+    },
+    walk: function(){
```

Comment:
if you return itself you can chain these methods

ex: phillip.sayName().saySomething()